WORKFLOW.md

```diff
@@ -0,0 +1,138 @@
+
+Fork repository:
+- `master` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests
+	- Tagged releases of Xroad-tests
+- `develop` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests 
+	- Country branch for test developme
+	- Latest country test development source code 
+	- Make pull request to develop for ria-ee/X-Road-tests
+
+## 6 Tagging
+
+6.1 The versions merged to X-Road-tests/master branch are tagged by Finnish / Estonian responsible persons.
+
+git tag -a 6.16.0 -m "X-Road-tests for Xroad version 6.16.0"
+
+The versions merged to X-Road-tests/develop branch are not tagged.
+
+## 7 Changelog management and Pull requests
+
+Changelog description is visible in pull requests. Pull requests from Partners made against the `ria-ee/X-Road-tests/master` or `ria-ee/X-Road-tests/develop` branch MUST follow these conventions:
+
+Pull request name format is 'country-month-year-description':
+Country=origin of the pull request
+Month=Month when the pull request was created
+Year=Year when the pull request was created
+Description=(short) Description of the pull request
+
+e.g Finnish-08-2017-Description-of-pull-request, Estonian-07-2017-Description-of-pull-request
+
+- Description field must contain at least the changelist. Any relevant additional information should also be provided here.
+
+## 8 Submitting and accepting tests work
+
+8.1 Upon completion of tests development, submit a pull request to `X-Road-tests/develop`.
+
+8.2	The pull request is reviewed by Finland and Estonia responsible persons. Additional reviewers can be added as necessary. The pull request is reviewed according to the acceptance criteria that was in effect when the work on this changeset started.
+
+8.2.1 Source code
+
+- Is the source code for tests and its dependencies available?
+
+8.2.2 CI build & tests
+- No merge conflicts
+- Test must be working LXC jenkins environment
+- Jenkins job is generated or added existing one
+- Test must working reliably e.g executed multiple times
+- If possible to make test independent
+
+8.2.3 Documentation
+- Is documentation updated?
+- Changelog:
+    - Version information
+    - Notes for changes
+
+8.2.4 Licensing
+- License exists in root folder
+
+8.2.5 Pull request
+- Pull requests are generally reviewed and accepted on first-come, first-served (FCFS) basis.Non-functional requirements
+- Descriptive git commit messages
+	- E.g "Added xroad-global-configuration test case 3.3" or "Fixed login test working with new layout"
+
+- New tests are made PR to Xroad public github develop branch
+    - In moving later Xroad development and Xroad tests at same time
+
+8.2.6 Browser acceptance testing
+- Browser versions
+    - Mozilla firefox 47.0.2
+    - Smoke tests with Chrome -> Not file download and upload, and certificates
+
+## 9 Bug fixes
+
+The Partner who created the latest version is main responsible of fixing the problem.
```

Comment:
As in https://github.com/vrk-kpa/xroad-joint-development/blob/master/CHARTER.md chapter 3 - Warranty

When an error is detected, the Partner who located the error is responsible of detecting the source of the error. Detecting the source of the error means tracking the first version of the software that contains the error. When the source is detected the Partner who created the version in question must be notified. The Partner who created the version is responsible of fixing the problem.

Reply:
changed